### ISSUE TITLE
Network timed out while trying to connect to https://index.docker.io/v1/repositories/library/[ANYTHING]. You may want to check your internet connection or if you are behind a proxy.

### ISSUE THREAD

**Issue by samlangdon** [docker/kitematic#1863]
### Expected behavior
Try to pull images from Docker Hub using either Powershell or Kitematic. Expect to see images downloading.

### Actual behavior
As per screenshots, the pull requests time out trying to connect to https://index.docker.io. 
![cap1](https://cloud.githubusercontent.com/assets/16637123/17260087/6448fc3e-55c6-11e6-8e73-c394b1a34d13.PNG)
![cap2](https://cloud.githubusercontent.com/assets/16637123/17260088/6468ef1c-55c6-11e6-8007-94085a140d7f.PNG)

### Information about the Issue
Running Docker on Windows 10 Pro 64 bit. My internet connection is functioning and to my knowledge, I'm not using a proxy. I was actually able to pull images earlier today via Powershell, then first noticed this problem after installing Kitematic.

### Steps to reproduce the behavior

  1. Open Windows Powershell, enter command: docker run --rm -v c:/Users:/data alpine ls /data
  2. Powershell responds with: Unable to find image 'alpine:latest' locally
Pulling repository docker.io/library/alpine
C:\Program Files\Docker\Docker\Resources\bin\docker.exe: Network timed out while trying to connect to https://index.dock
er.io/v1/repositories/library/alpine/images. You may want to check your internet connection or if you are behind a proxy

Also: 
  1. Open Kitematic, repos in hub are listed.
  2. Choose Create to pull any repo
  3. After a while, error is returned: Network timed out while trying to connect to https://index.docker.io....

**Comment by henrylle** [comment]
I have the same error.

**Comment by anggun-indra** [comment]
i have same error too

**Comment by SriramKeerthi** [comment]
Same error. I don't see any progressbars either but that's probably because it can't connect in the first place. I also see a 404 if I restart Kinematic.

**Comment by FrenchBen** [comment]
I'm able to pull all images from the homepage - If you start Kitematic from PowerShell, do you have the same issue?

**Comment by henrylle** [comment]
In my case the problem was another network driver, like VirtualBox. After uninstall VirtualBox and reinstall Docker and Kitematic, everything works!

**Comment by SriramKeerthi** [comment]
Didn't work through the shell either for me, I suspect it was a network issue caused during installation. Restarting my VPN connection fixed it.

**Comment by clayderhua** [comment]
me too. I have the same error.

**Comment by samlangdon** [comment]
I eventually fixed it by uninstalling my network card - it reinstalled automatically after a bit, then I restarted Docker. I think it was something to do with the Hyper V virtual switch for Docker (DockerNat) going awry, as I noticed I had several DockerNat's setup after a few restarts, but reinstalling my network card seemed to be the easiest fix. Hope that helps someone!

**Comment by mishrsud** [comment]
I experienced the same issue, My config:
Windows 10 Pro x64, v 1607, build 14393.576. I had virtual box installed, so following the suggestions here and elsewhere on forums, I uninstalled it, uninstalled Docker, rebooted and the installed Docker again - no luck. This was with the latest stable Docker for Windows downloaded 31/12/2016.
However, when I uninstalled the latest stable and installed the latest beta (1.13.0-rc4-beta34 (9562)) everything worked fine!
